network/endpoint_test.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Microsoft. All rights reserved.
+// MIT License
+
+package network
+
+import (
+	"testing"
+)
+
+func TestGetPodName(t *testing.T) {
+	testData := map[string]string{
+		"nginx-deployment-5c689d88bb":       "nginx",
```

Comment:
Can you gofmt this? It's just spacing/formatting. Unsure if it looks like this because of GitHub